### ISSUE TITLE
Expose UserManager

### ISSUE THREAD

**Issue by GordonBlahut** [perarnborg/vuex-oidc#138]
Is it possible to expose access to the UserManager for lower level access (even if it's marked as not-recommended/at your own risk, etc.)? For example, to workaround a limitation in oidc-client where the claims are not refreshed after silent renew for clients using refresh tokens:
https://github.com/IdentityModel/oidc-client-js/issues/1034

**Comment by GordonBlahut** [comment]
That's right. My understanding is that it's better not to have multiple UserManager instances at the same time so it would be ideal to get a reference to the same instance.

**Comment by zablik** [comment]
+1 on this ability.
I don't this now a can call UserManager.signinRedirect() and UserManager.signoutRedirect()

**Comment by perarnborg** [comment]
Just wanted to state that I am planning to implement this, but I have not found the time for it.
